android/app/src/main/res/values-ro/strings.xml

```diff
@@ -1498,7 +1498,8 @@
     <string name="type.place.islet">Insult</string>
     <string name="type.place.isolated_dwelling">Locuință izolată</string>
     <string name="type.place.locality">Localitate</string>
-    <string name="type.place.neighbourhood">Cartier</string>
+    <string name="type.place.quarter">Cartier</string>
+    <string name="type.place.neighbourhood">Vecinătate</string>
```

Comment:
I'm not sure about these Romanian translations. Sure, "quarter" and "cartier" are etymologically related, and "neighbourhood" and "vecinătate" both come from words meaning "neighbour". But if you ask a native Romanian speaker to translate "neighbourhood" they'll say "cartier", and "[vecinătate](https://dexonline.ro/definitie/vecinătate)" has no geographical meaning.

I'm not aware of any term meaning "named part of a city bigger than a neighbourhood". I see other Latin languages have the same translation for both (`it`, `pt`), I suggest we use "cartier" for both "quarter" and "neighbourhood".

Reply:
Yeap, this suburb > quarter > neighbourhood hierarchy is not customary for some languages / cultures, hence its hard to to translate concisely.
I think its fine to have identical translations for e.g. quater and neighbourhood.
However one trick that might work is to use e.g. a "smaller cartier" :) for neighbourhood (if it would make any sense in Romanian).

Reply:
I looked for synonyms and *if we must* have different translations we could use "cvartal" or "microraion", but they are not in common usage so I keep my suggestion to go with "cartier" for both.

Looking at some random cities in Romania on OSM, it looks like the subdivisions are either `suburb` or `neighbourhood`. Is there a way to interogate an area and find all features with a certain `place` value?

Reply:
> Is there a way to interogate an area and find all features with a certain `place` value?

You can check via overpass-turbo, e.g. https://overpass-turbo.eu/s/1WMP